lib/spack/spack/cmd/common/arguments.py

```diff
@@ -72,6 +73,35 @@ def _specs(self, **kwargs):
         return sorted(specs.values())
 
 
+class SetParallelJobs(argparse.Action):
+    """Sets the correct value for parallel build jobs.
+
+    The value is pushed on top of the current configuration scopes, and can be
+    retrieved using the spack.config API.
+    """
+    def __call__(self, parser, namespace, values, option_string):
+        # Values is a single integer, type conversion is already applied
+        # see https://docs.python.org/3/library/argparse.html#action-classes
```

Comment:
I used the same names and signature of `argparse.Action`, so that they resonate with anybody who's used to that package and looks at docs. Not opposed to change names if you think it's better. In case which one you prefer?

Reply:
Jobs :)